conans/test/integration/private_deps_test.py

```diff
@@ -128,100 +128,61 @@ def consumer_private_test(self):
 
     def reuse_test(self):
```

Comment:
This test was doing too much, lots of useless repetition. Simplified in Win it goes from 100secs => 55secs